edgelet/build/charts/edge-kubernetes/templates/iotedged-deployment.yaml

```diff
@@ -56,11 +61,20 @@ spec:
       - name: config
         secret:
           secretName: {{ include "edge-kubernetes.fullname" . }}-iotedged-config
+      {{- if .Values.iotedged.certificates }}
+      {{- /*
+        NOTE: This sets up iotedged with a volume "/etc/edgecerts" populated with the given CA certs.
+      */}}
+      - name: edge-certs
+        secret:
+          secretName: {{ include "edge-kubernetes.fullname" . }}-edgecerts
+      {{else}}
       {{- /*
         NOTE: This sets up iotedged in the "quickstart" mode which is NOT meant
         for production use. For PoCs though, its wonderful! For production, this
         volume should probably be populated with proper CA certs.
       */}}
+      {{end}}
```

Comment:
Nit: Is the note about being in "quickstart" mode still relevant? Maybe we should change this comment to indicate that if you don't want to run in quickstart mode then you should specify certs via `values.yaml`.